### PR TITLE
73483944 limit scale of miles

### PR DESCRIPTION
**Pull request by cdemyanovich**

### CODE REVIEW

File: src/data/listings.coffee [hunk 1]
```diff
@@ -95,7 +95,7 @@ define [
         latitude: data.latitude
         lng: data.longitude
         longitude: data.longitude
-        miles: mapUtils.limitScaleOf(distanceConversion.convertMetersToMiles(data.radius))
+        miles: mapUtils.limitScaleOf(distanceConversion.convertMetersToMiles(data.radius), 1)
```
Comment (c0) [inline src/data/listings.coffee]: Should miles be rounded? So 10.5 becomes 11?

Comment (cdemyanovich) [inline src/data/listings.coffee]: Makes sense to me. Any concerns with doing that, @pasha?

Comment (c0) [inline src/data/listings.coffee]: Oh, the precision is 1, not 0. Does 1/10 of a mile make enough difference to round or not?

Comment (pasha) [inline src/data/listings.coffee]: Sounds good top me, lets do it.

**Comment by cdemyanovich** [comment]
We now round miles to the nearest integer. But, I left the change to `limitScaleOf`, since it makes the function more flexible.

**Comment by c0** [comment]
:+1:

**Pull request merged**